DetDescendantReport-images/detdescendantreporti.py

```diff
@@ -14,6 +14,7 @@
 # Copyright (C) 2011      Tim G L Lyons
 # Copyright (C) 2013-2014 Paul Franklin
 # Copyright (C) 2017      Jon Schewe <jpschewe@mtu.net> - modifications to add all images
+# Copyright (C) 2022      Gary Griffin - added event media
```

Comment:
With author history permanently recorded in GitHub (along with exact changes made by the author per commit), adding attribution here is superfluous.

Reply:
When I rewrote a plugin (GetGOV) years ago and did not update the copyright, I was told that I should. I was just following that previous advise.

Reply:
@hgohel  I too have been adding this copyright, please let us know what is the preference for updating this information (or not) so that the PR can be merged.

Reply:
This comment is also not marked as change required - don't let this stop the PR from being committed. For clarification I [created a discussion thread](https://gramps.discourse.group/t/updating-copyright-authorship-and-reason-for-change-in-source-code-required/2592) so maintainers can help clarify expectations.